Source/Atomic/UI/UIEditField.cpp

```diff
@@ -213,6 +215,17 @@ void UIEditField::OnFocusChanged(bool focused)
             if (!w->GetMultiline())
                 styleEdit->selection.SelectAll();
             firstFocusFlag_ = true;
+
+#if defined(ANDROID) || defined( __ANDROID__) || defined(IOS)
+
```

Comment:
I will use those instead, I found the current set of defines by grepping in the Sources/Atomic directory.